nginx.tmpl

```diff
@@ -18,11 +22,33 @@ http {
     location ~* ^/images/(full|thumbs)(.*) {
       set $s3_bucket        's3.eu-central-1.amazonaws.com/${IMAGE_STORAGE_NAME}';
       set $url_image        '$1$2';
+      set $width -;
+      set $height -;
+
+      if ($args ~ width=([0-9]+)) {
+        set $width $1;
+      }
+
+      if ($args ~ height=([0-9]+)) {
+        set $height $1;
+      }
+
+      image_filter_buffer 20M;
+      image_filter_jpeg_quality 75;
```

Comment:
Ah. ok. Lurte på hvorfor vi ikke satt 100, men hvis 75 er anbefalt, så kjører vi gjerne med det. Noen begrunnelse for hvorfor ikke 100 er anbefalt? Fordi det egentlig ikke er mulig å ha 100% kvalitet på et skalert bilde?